hadoop-ozone/cli-admin/src/main/java/org/apache/hadoop/hdds/scm/cli/container/ContainerIDParameters.java

```diff
@@ -31,4 +32,34 @@ public class ContainerIDParameters extends ItemsFromStdin {
   public void setContainerIDs(List<String> arguments) {
     setItems(arguments);
   }
+
+  public List<Long> getValidatedIDs() {
+    List<Long> containerIDs = new ArrayList<>(size());
+    boolean allValid = true;
+
+    for (String input: this) {
+      boolean idValid = true;
+      try {
+        long id = Long.parseLong(input);
+        if (id <= 0) {
+          idValid = false;
+        } else {
+          containerIDs.add(id);
+        }
+      } catch (NumberFormatException e) {
+        idValid = false;
+      }
+
+      if (!idValid) {
+        allValid = false;
+        System.err.println("Container ID must be a positive integer, got: " + input);
+      }
+    }
+
+    // After errors have been printed for all invalid containers, exit PicoCLI with a non-zero result.
+    if (!allValid) {
+      throw new IllegalArgumentException();
```

Comment:
Can we add an exception message here so that we know why this exception is being thrown?

Reply:
A message to stderr has already been printed for each invalid container, this is just to make picoCLI exit after we have finished the list. If we want a summary message of some kind we could put it here, but the user will get feedback on each error anyways.

Reply:
I've added an error message for checking status of open containers and tests for this as well.